packages/theme-patternfly-org/templates/mdx.js

```diff
@@ -69,7 +69,7 @@ const MDXChildTemplate = ({
       {toc.length > 1 && (
         <TableOfContents items={toc} />
       )}
-      <div className={katacodaLayout? "ws-mdx-content-katacoda" : "ws-mdx-content"}>
+      <div className={katacodaLayout? "ws-mdx-content-katacoda" : "ws-mdx-content pf-m-fill"}>
```

Comment:
> `pf-m-fill`

This works on some component/layout elements (like `.pf-c-page__main-section`, or a child of `.pf-l-split`), but doesn't appear to be doing anything here, so not sure if you need it.